app/scripts/modules/amazon/loadBalancer/details/loadBalancerDetail.controller.js

```diff
@@ -88,7 +90,7 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.details.controller',
       // If the user navigates away from the view before the initial extractLoadBalancer call completes,
       // do not bother subscribing to the refresh
       if (!$scope.$$destroyed) {
-        app.onRefresh($scope, extractLoadBalancer);
+        app.loadBalancers.onRefresh($scope, extractLoadBalancer);
```

Comment:
Yeah, might as well convert it, since we will need to do so when we cut this controller over to TS...